test/features/install_explainability.feature

```diff
@@ -0,0 +1,56 @@
+@explainability
+@kafka
+Feature: Kogito Explainability service
+
+  Background:
+    Given Namespace is created
+
+  @smoke
+  # Disabled as long as https://issues.redhat.com/browse/KOGITO-3176 is not solved
+  @disabled
+  Scenario: Install Kogito Explainability
+    Given Kogito Operator is deployed with Kafka operator
+    And Install Kogito Explainability with 1 replicas
+    Then Kogito Explainability has 1 pods running within 10 minutes
+    And HTTP GET request on service "explainability" is successful within 2 minutes with path "health/live"
```

Comment:
I think that if this endpoint returns something is down, the pod never starts and hence the previous step should be sufficient.